airbyte-integrations/connectors/source-stripe/unit_tests/resource/http/response/persons_event_created.json

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
nit: In the test I've done, I would usually avoid creating one file per event type to reduce maintenance (see https://github.com/airbytehq/airbyte/pull/33306/files#diff-00bd50afcc672f2737548c1ff387d3a09094e02e611e0dde8138898acdbab374R338-R339). That way, if the events response change a bit, we only have to change the template and the rest should stay as is. Let me know what you think about this